Core/Core/View/Base/FullScreenErrorView/FullScreenErrorView.swift

```diff
@@ -10,10 +10,15 @@ import Theme
 
 public struct FullScreenErrorView: View {
     
-    public enum ErrorType {
+    public enum ErrorType: Equatable {
         case noInternet
         case noInternetWithReload
         case generic
+        case noVideos(error: String, image: SwiftUI.Image)
+        case noHandouts(error: String, image: SwiftUI.Image)
+        case noAnnouncements(error: String, image: SwiftUI.Image)
+        case noCourseDates(error: String, image: SwiftUI.Image)
+        case noCourseware(error: String, image: SwiftUI.Image)
```

Comment:
We are passing the same data in all the cases, how about defining a single case for all the cases and use that? Thoughts?

Reply:
I was thinking the same thing, but I opted for this approach to accommodate any future type-specific UI requirements. We can opt for single case for now. Thoughts?

Reply:
Opting for a single case is good at the moment. We will update it in the future as per the need.